src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/BallotPanelPoolReportITest.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/mod/truncate.sql",
+    "/db/mod/reports/CurrentPoolStatusReportControllerITest_typical.sql"
+})
+class BallotPanelPoolReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public BallotPanelPoolReportITest(TestRestTemplate template) {
+        super(template, BallotPanelPoolReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+                                 .poolNumber("415230103")
+                                 .build());
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")//False positive
```

Comment:
@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")//False positive

Should be at class level instead of method level (Means you don't need to have this on all test classes)